source/jormungandr/jormungandr/scenarios/simple.py

```diff
@@ -148,6 +148,7 @@ def pt_objects(self, request, instance):
         req.requested_api = type_pb2.pt_objects
         req.pt_objects.q = request['q']
         req.pt_objects.depth = request['depth']
+        req.pt_objects.disable_geojson = request['disable_geojson']
```

Comment:
At first I wanted to sugest the usage of `get()` to prevent an error in case of the key `disable_geosjon` not being set in the request dict, this should not happens since request parser always fill it. But we don't do it for others attributes and it will require to duplicate all the default values. 

So I don't know what to do, the request parser only handle http request, if we do an internal call we still have to check for the key. I think there is a lot of refactoring to do on this part of jormungandr: Resources, instance manager, instances, scenarios are complex and a lot of thing are duplicate in theses. So I think I will just add this to my todo list